File: 3-bimestre/Kevin-Vitor/CPP/ex05.cpp
/*
 ETEC VASCO ANTONIO VENCHIARUTTI
    Autor: Kevin Silva
    Turma: 1ºc2

    DESCRIÇÃO: Leia um número inteiro de 1 a 10 e exiba sua tabuada. Caso o número esteja fora do intervalo, solicite novamente até que o valor seja válido.
    
*/

#include <iostream>
using namespace std;

int main()
{
    int numero;
    int contador = 1;

    cout << "Digite um numero de 1 a 10: ";
    cin >> numero;

    while (numero < 1 || numero > 10)
    {
        cout << "Valor invalido. Digite novamente: ";
        cin >> numero;
    }

    while (contador <= 10)
    {
        cout << numero << " x " << contador << " = " << numero * contador << endl;
        contador++;
    }

    return 0;
}

File: 3-bimestre/Kevin-Vitor/PYTHON/ex05.py
'''
    ETEC VASCO ANTONIO VENCHIARUTTI
    Autor: Kevin Silva
    Turma: 1ºc2

    DESCRIÇÃO:  Lê um número de 1 a 10 e exibe sua tabuada, validando a entrada.

'''
#exibe a mensagem e lê a resposta do usuario
num = int (input("digite um número de 0 a 10: "))
contador = 0
#se o número não estiver entre 1 e 10 o programa mostrara a mensagem dentro do imput
while num < 1 or num > 10 :
    num = int (input ("Número invalido, por favor, digite um número de 1 a 10: "))
#apresentara a tabuada do número digitado
while contador <= 10:
        print(f"{num} x {contador} = {num} * {contador}")
        contador += 1